tests/validation/phone-validator-adapter.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { PhoneValidatorAdapter } from '../../src/application/validation/validators'
```

Comment:
Remove "../../../src/" from import, because the `src` folder is already set as the root of the project in typescript